### PR TITLE
Invalidate all sessions

### PR DESCRIPTION
**Pull request by alexurguz**
## Summary

Implemented controller rinvalidate sessions endpoint post /users/sessions/invalidate_all
Implemented validations
Implemented tests
Implemented documentation

## Known Issues

None

## Trello Card

[Invalidate all sessions](https://trello.com/c/RdVgFOCi/24-deshabilitar-todas-las-sesiones)

### CODE REVIEW

**Comment by martinremis** [comment]
Change the Pull Request base to #18 , in order to understand the new changes. It is really hard to check like this :sweat_smile:

File: app/controllers/users.js [hunk 1]
```diff
@@ -52,6 +53,9 @@ const signIn = async (req, res, next) => {
 
     logger.info(`user with email ${user.email} has logged in successfully`);
 
+    const savedToken = await tokenService.createToken(userSerialized, token);
+    logger.info(`users-controller::signIn::TokenSaved: ${JSON.stringify(savedToken)}`);
```
Comment (martinremis) [inline app/controllers/users.js]: Do not log tokens or security information

File: app/routes.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ exports.init = app => {
   app.get('/health', healthCheck);
 
   app.post('/signin', signInValidator, userController.signIn);
+  app.post('/users/sessions/invalidate_all', validateToken, userController.invalidateAllSessions);
```
Comment (martinremis) [inline app/routes.js]: For routes it is preferable to use kebab-case instead o snake_case

File: app/services/tokens.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+const { Token } = require('../models');
+const logger = require('../logger');
+const errors = require('../errors');
+
+const createToken = async (userData, token) => {
+  try {
+    logger.info(`tokens-service::createToken::userData::${JSON.stringify(userData)}token::${token}`);
```
Comment (martinremis) [inline app/services/tokens.js]: Do not log tokens

Comment (martinremis) [reply app/services/tokens.js]: Or passwords (if the userData has the field)

**Review by martinremis — CHANGES_REQUESTED** [review]

File: app/controllers/users.js (hunk 1 above)
Comment (alexurguz) [reply app/controllers/users.js]: Ready!

File: app/routes.js (hunk 2 above)
Comment (alexurguz) [reply app/routes.js]: Ready!

File: app/services/tokens.js (hunk 3 above)
Comment (alexurguz) [reply app/services/tokens.js]: Ready!

**Review by martinremis — APPROVED** [review]

**Pull request merged**